frontend/public/i18n.js

```diff
@@ -59,6 +59,8 @@ i18n
       'rhoas-plugin',
       'topology',
       'tour',
+      'webhook-receiver-form',
+      'yaml',
```

Comment:
Can you please remove `webhook-receiver-form` and `yaml` from `i18n.js`?

Reply:
Removed